tests/functional/incremental-fetching-empty-rows/setUp.php

```diff
@@ -10,4 +10,5 @@
 
     // Auto increment table
     $manager->createAITable();
+    $manager->addAIConstraint();
 };
```

Comment:
Tento test nemal `constraint`, dost dlho som hladal, preco to tu nejde.
Tak som to tam rovno pridal.